smac/facade/func_facade.py

```diff
@@ -18,9 +18,9 @@
 
 def fmin_smac(func: typing.Callable,
               x0: typing.List[float],
-              bounds: typing.List[typing.List[float]],
-              maxfun: int=-1,
-              rng: np.random.RandomState=None,
+              bounds: typing.List[typing.Iterable[float]],
+              maxfun: int = -1,
+              rng: typing.Union[np.random.RandomState, int] = None,
               scenario_args: typing.Mapping[str, typing.Any]=None,
```

Comment:
If you add spaces for the lines above, you should also add them here :-)
scenario_args: typing.Mapping[str, typing.Any]X=XNone

Reply:
Done